src/frontend/src/pages/Calendar/SummaryModal/DesignReviewSummaryModal.tsx

```diff
@@ -0,0 +1,92 @@
+import { DesignReview } from 'shared';
+import NERModal from '../../../components/NERModal';
+import { Box, Grid, IconButton } from '@mui/material';
+import EditIcon from '@mui/icons-material/Edit';
+import SaveIcon from '@mui/icons-material/Save';
+import { useState } from 'react';
+import { DesignReviewDelayModal } from './SummaryComponents/DesignReviewDelayModal';
+import StageGateWorkPackageModalContainer from '../../WorkPackageDetailPage/StageGateWorkPackageModalContainer/StageGateWorkPackageModalContainer';
+import DesignReviewSummaryModalDetails from './SummaryComponents/DesignReviewSummaryModalDetails';
+import DesignReviewSummaryModalCheckBox from './SummaryComponents/DesignReviewSummaryCheckbox';
+import DesignReviewSummaryModalButtons from './SummaryComponents/DesignReviewSummaryModalButtons';
+import DesignReviewSummaryModalAttendees from './SummaryComponents/DesignReviewSummaryModalAttendees';
+import { getTeamTypeIcon } from '../../CalendarPage/CalendarComponents/CalendarDayCard';
+import DesignReviewSummaryModalDetailsButton from './SummaryComponents/DesignReviewDelayModalDetailsButton';
+
+interface DRCSummaryModalProps {
+  open: boolean;
+  onHide: () => void;
+  designReview: DesignReview;
+}
+
+const DRCSummaryModal: React.FC<DRCSummaryModalProps> = ({ open, onHide, designReview }) => {
+  const [checked, setChecked] = useState<boolean>(false);
+  const [showStageGateModal, setShowStageGateModal] = useState<boolean>(false);
+  const [showDelayModal, setShowDelayModal] = useState<boolean>(false);
+  const [editing, setEditing] = useState<boolean>(false);
+
+  return (
+    <NERModal
+      open={open}
+      onHide={onHide}
+      title={designReview.wbsName}
+      hideFormButtons
+      icon={getTeamTypeIcon(designReview.teamType.teamTypeId, true)}
+    >
+      <Box width="500px">
+        <Box position="absolute" right="16px" top="12px">
+          {editing ? (
+            <IconButton onClick={() => setEditing(false)}>
+              <SaveIcon />
+            </IconButton>
+          ) : (
+            <IconButton onClick={() => setEditing(true)}>
+              <EditIcon />
+            </IconButton>
+          )}
+        </Box>
+        <StageGateWorkPackageModalContainer
+          wbsNum={designReview.wbsNum}
+          modalShow={showStageGateModal}
+          handleClose={() => setShowStageGateModal(false)}
+          hideStatus
+        />
+        <DesignReviewDelayModal open={showDelayModal} onHide={() => setShowDelayModal(false)} designReview={designReview} />
+
+        <Grid container direction="column">
```

Comment:
this is just a vertical stack, so you should be able to remove all this grid stuff, and just put one Box around all the components from here to line 86. Also remove any empty lines between component tags, if you set a file up right, you should be able to read it easily without the spaces